unskript-ctl/unskript_ctl_notification.py

```diff
@@ -205,6 +208,8 @@ def create_script_summary_message(self, output_metadata_file: str):
                     </tr>
                 </table>
         '''
+
+        self.logger.info(message)
```

Comment:
remove this

Reply:
This will be handled via https://unskript.atlassian.net/browse/EN-5240